Frontend/ui-driver/src/Flow.purs

```diff
@@ -2369,7 +2369,10 @@ homeScreenFlow = do
                 let payerVpa = fromMaybe "" response.payerVpa
                 modifyScreenState $ HomeScreenStateType (\homeScreen -> homeScreen 
                   { data {
-                      activeRide {endOdometerReading = (\(API.OdometerReading {value}) -> value) <$> response.endOdometerReading}, 
+                      activeRide {
+                        endOdometerReading = (\(API.OdometerReading {value}) -> value) <$> response.endOdometerReading,
+                        parkingCharge = if isSpecialPickUpZone && (getMerchant FunctionCall) == YATRISATHI then response.parkingCharge else Nothing
```

Comment:
Why merchant check
If required Add a config and then use, Remove this merchant check